config/routes.rb

```diff
@@ -1,4 +1,24 @@
 Rails.application.routes.draw do
+
+
+  root :to => "articles#index"
+
+  get "showarticle" => "articles#showarticle"
```

Comment:
and why are you defining the same route twice? make it just once... and keep the one inside the resources, that is how we do.